src/java/org/apache/cassandra/service/DataResolver.java

```diff
@@ -293,22 +310,30 @@ private boolean isReversed(List<UnfilteredRowIterator> versions)
             return false;
         }
 
+        @VisibleForTesting
+        void waitForRepairResults() throws TimeoutException
+        {
+            FBUtilities.waitOnFutures(repairResults, DatabaseDescriptor.getWriteRpcTimeout(), finishedRepairs);
```

Comment:
I extracted this call to separate method so I can override its behavior in the test.